howtos/diversification.rst

```diff
@@ -0,0 +1,37 @@
+Diversification of Search Results
+=================================
+
+.. contents:: Table of contents
+
+Introduction
+------------
+
+Xapian allows for diversification of documents which are stored in the form of an MSet.
+This feature is a well-known technique in information retrieval used to increase
+user satisfaction, especially for ambiguous queries.
+
+Xapian currently has an implementation of an *implicit* method (using documents as features,
+as opposed to using query based features such as query logs) adapted from the C :sup:`2` - GLS method mentioned in Scalable and Efficient Web Search Results Diversification, Naini et al. 2016. This saves the cost of not having to provide external features such as query
+logs, while still achieving the desired diversification effect, which according to
+the paper is reasonable enough for practical uses as tested on the public data set - ClueWeb09 with TREC Web 09/10 queries.
+
+API
+---
+ 
+Diversification on an MSet of results can be achieved by using the
+:xapian-method:`Diversify` class, e.g.::
```

Comment:
Surely `xapian-class` not `xapian-method`?

Reply:
I've pushed a fixup commit for this.